pkg/trivy/wrapper.go

```diff
@@ -125,7 +125,7 @@ func (w *wrapper) parseScanReports(reportFile io.Reader) (report ScanReport, err
 
 	// Collect all vulnerabilities to single scanReport to allow showing those in Harbor
 	report.Target = scanReports[0].Target
-	report.Vulnerabilities = []Vulnerability{}
+	report.Vulnerabilities = make([]Vulnerability, 0, len(scanReports[0].Vulnerabilities))
```

Comment:
We know that there are at least `scanReports[0].Vulnerabilities` number of vulnerabilities so we can pre-allocate some memory.